roles/confluent.common/tasks/host_validations.yml

```diff
@@ -1,22 +1,37 @@
 ---
+- name: Confirm Running as Root
+  assert:
+    that: ansible_user_id == 'root'
+    fail_msg: >-
+      CP-Ansible must have sudo access. Current User ID: {{ansible_user_id}} is not root.
+      Make sure to set ansible_become to true and provide sudo access.
+      To skip host validations, set validate_hosts to false.
+  tags: validate
```

Comment:
@ldom Added you as a reviewer bc this task will need a conditional in your branch